### DISCUSSION TITLE
OpenHue Roadmap

### DISCUSSION THREAD

**Discussion by thibauult** [thibauult/open-hue#11] [Announcements]
# OpenHue API Roadmap 🛣️
This roadmap outlines the planned additions and enhancements to the OpenHue OpenAPI specification.
Our goal is to continually improve and expand the functionality of OpenHue to better serve developers and the Philips Hue community.
Below, you'll find a list of upcoming APIs that we plan to include.

> ℹ️ Please note that the APIs below are ordered by priority.

- [x] `/api`
- [x] `/resource`
- [x] `/resource/light`
- [x] `/resource/bridge`
- [x] `/resource/bridge_home`
- [x] `/resource/scene`
- [x] `/resource/smart_scene`
- [x] `/resource/room`
- [x] `/resource/zone`
- [x] `/resource/light_level`
- [x] `/resource/grouped_light`
- [x] `/resource/temperature`
- [x] `/resource/device`
- [x] `/resource/device_power`
- [ ] `/resource/geofence_client`
- [ ] `/resource/geolocation`
- [x] `/resource/motion`
- [ ] `/resource/contact`
- [ ] `/resource/tamper`
- [ ] `/resource/button`
- [ ] `/resource/entertainment_configuration`
- [ ] `/resource/entertainment`
- [ ] `/resource/camera_motion`
- [ ] `/resource/relative_rotary`
- [ ] `/resource/behavior_script`
- [ ] `/resource/behavior_instance`
- [ ] `/resource/homekit`
- [ ] `/resource/matter`
- [ ] `/resource/matter_fabric`
- [ ] `/resource/zigbee_connectivity`
- [ ] `/resource/zgp_connectivity`
- [ ] `/resource/zigbee_device_discovery`

## Contributing to the Roadmap

We encourage contributions and suggestions from the OpenHue community.
If you have ideas for new APIs, features, or improvements, please [open an issue](https://github.com/openhue/openhue-api/issues) on our GitHub repository.
Your feedback is valuable in shaping the future of OpenHue.
Stay tuned for updates and announcements as we work on these exciting additions and enhancements.
We look forward to delivering a more robust and feature-rich OpenHue experience.

**Comment by thibauult** [comment]
➕ The `scene` API is now supported via https://github.com/openhue/openhue-api/pull/13. 

📦 Release https://github.com/openhue/openhue-api/releases/tag/0.4

**Comment by thibauult** [comment]
➕ The `room` API is now supported via https://github.com/openhue/openhue-api/pull/14. 

📦 Release https://github.com/openhue/openhue-api/releases/tag/0.5

**Comment by thibauult** [comment]
➕ The `light_level`, `grouped_light ` and `temperature ` APIs are now supported! 

📦 Release https://github.com/openhue/openhue-api/releases/tag/0.7

**Comment by jonathancross** [comment]
It's been almost 2 years, is this roadmap still a thing?

**Reply by thibauult** [reply]
Hi @jonathancross - absolutely! I just forgot to update it as we recently added the [`/smart_scene` API](https://github.com/openhue/openhue-api/pull/35). It is now totally up-to-date according to which APIs have been added to the spec so far 👍
